src/chat_do.py

```
@@ -0,0 +1,406 @@
+        print(f"[ChatDO.on_webSocketError] error={error!r}")
+
+    def _broadcast(self, payload, exclude_session_id=None):
+        for sid, info in self.sessions.items():
+            if sid == exclude_session_id:
+                continue
+            try:
+                info["ws"].send(payload)
+            except Exception as exc:
+                print(f"[ChatDO._broadcast] sid={sid} user_id={info.get('user_id')} error={exc!r}")
+
+    async def _ensure_table(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_ensure_table`

Add return type annotation: `None`

(ANN202)